fluent/syntax/ast.py

```diff
@@ -73,7 +73,7 @@ def visit(value):
 
         return fun(node)
 
-    def equals(self, other, with_spans=False):
+    def equals(self, other, ignored_fields=['span']):
```

Comment:
AFAIK, the pythonic way of doing this is to make the default None, and then at the top of the body do a

    if ignored_fields is None:
        ignored_fields = ['span']

Not sure why exactly, but that's what I've been told about non-trivial things as default arguments.